static/cli.ts

```diff
@@ -1,2 +1,2 @@
-console.log('\x1b[31mERROR\x1b[0m Stencil for Deno has reached end-of-life')
+console.log('\x1b[31mERROR\x1b[0m Stencil for Deno has reached end-of-life');
```

Comment:
We still keep this around for Deno support - specifically to say it's EoL

Reply:
We have support for Deno? 🤔

Reply:
We did, a few years ago. That's part of the reason for the `sys`/`platform` abstraction in Stencil - one for Node, one for Deno, one for the web